agent/structs/system_metadata.go

```diff
@@ -0,0 +1,36 @@
+package structs
+
+// SystemMetadataOp is the operation for a request related to system metadata.
+type SystemMetadataOp string
+
+const (
+	SystemMetadataOpUpsert SystemMetadataOp = "upsert"
+	SystemMetadataOpDelete SystemMetadataOp = "delete"
```

Comment:
minor, to align more with the KVOp naming
```suggestion
	SystemMetadataUpsert SystemMetadataOp = "upsert"
	SystemMetadataDelete SystemMetadataOp = "delete"
```

Reply:
done